README.md

```diff
@@ -133,10 +133,10 @@ npm i
 npm run certs:make
 
 # Build the SDK and watch for changes
-npm start
+npm run watch
```

Comment:
Making some adjustments as these scripts get more and more numerous. I'm trying to avoid changing stuff like this has it can confuse those already using the SDK when the old script is removed, but to move forward, I felt we need to leverage namespacing more.